Xwt.Gtk.Mac/GtkMacOpenFileDialogBackend.cs

```diff
@@ -46,7 +46,7 @@ public void Initialize (System.Collections.Generic.IEnumerable<FileDialogFilter>
 			if (!string.IsNullOrEmpty (initialFileName))
 				this.DirectoryUrl = new NSUrl (initialFileName, true);
 
-			this.Prompt = "Select File" + (multiselect ? "s" : "");
+			this.Prompt = Application.TranslationCatalog.GetPluralString("Select File", "Select Files", multiselect ? 2 : 1);
```

Comment:
Lets merge this as it is, if we like the approach and think about translation optimization and plurals separately.